src/js/angular/clustermanagement/templates/cluster-list.html

```diff
@@ -0,0 +1,124 @@
+<div onto-loader-new ng-if="loader" class="create-cluster-loader" message="getLoaderMessage" size="100  "></div>
+
+<div class="table-responsive" ng-class="isEditMode() ? 'edit-mode': 'preview-mode'">
+    <div class="pull-right mb-1">
+        <button ng-click="addNode()" ng-disabled="editedNodeIndex !== undefined"
+                class="btn btn-primary add-node-btn"
+                gdb-tooltip="{{'cluster_management.update_cluster_group_dialog.actions.add_node_tooltip' | translate}}">
+            <em class="icon-plus"></em>{{'cluster_management.update_cluster_group_dialog.actions.add_node' | translate}}
```

Comment:
You can use the font awesome icons already. Donka prefers the `regular` once.

Reply:
done